src/conversion/modelconversiondispatcher.js

```diff
@@ -68,11 +83,11 @@ import extend from '@ckeditor/ckeditor5-utils/src/lib/lodash/extend';
  *			// Add the newly created view element to the view.
  *			viewWriter.insert( viewPosition, viewElement );
  *
- *			// Remember to stop the event propagation if the data.item was consumed.
+ *			// Remember to stop the event propagation.
  *			evt.stop();
  *		} );
  *
- * Callback that "overrides" other callback:
+ * Callback that overrides other callback:
```

Comment:
I would remove this sample, it is too complicated, after reading it I understand less than before. Instead you could add a regular convresion for attribute->wrap (like bold).